README.md

```diff
@@ -37,45 +39,62 @@ NOTE: Currently, to access the API, you must be connected to the McGill network.
+export $(cat .env | xargs)
 ```
 
-To set environment variables in Windows from CMD:
+The below instructions for Docker and Python assume that you have at least set `USERNAME` and `PASSWORD` in your current environment.
 
-```bash
-$ set KEY=value
+### Docker
+First, [install docker](https://docs.docker.com/get-docker/).
+
+ You can then run a Docker container for the API in three ways:
+#### Option 1: Use the `docker-compose.yaml` file
+
+First, [install docker-compose](https://docs.docker.com/compose/install/).
 
-# For example
-$ set USER=someuser
+If needed, update your .env file with optional environment variables for the docker-compose configuration:
+- `API_TAG`: Tag for API Docker image (default: `latest`)
+- `DOG_ROOT`: container name or IP address for the graph database (default: `graph`)
+- `STARDOG_TAG`: Tag for Stardog Docker image (default: `7.7.3-java11-preview`)
+- `STARDOG_ROOT`: Path to directory on host machine containing a Stardog license file (default: `~/stardog-home`)
+
+Note: To avoid conflicts related to [Docker's environment variable precedence](https://docs.docker.com/compose/environment-variables/envvars-precedence/), ensure that any variables defined in your `.env` file are not already set in your current shell environment with **different** values.
+
+Then spin up the containers using Docker Compose:
+```bash
+docker compose up -d
 ```
-The below instructions for Docker and Python assume that you have already set `USER` and `PASSWORD` in your current environment.
 
-### Docker
-Follow the [official documentation](https://docs.docker.com/get-docker/) for installing Docker. You can then run a Docker container for the API in two ways:
-#### Option 1: Pull the latest image from Docker Hub
+#### Option 2: Use the latest image from Docker Hub
 ```bash
+source .env # set your environment variables 
 docker pull neurobagel/api
-docker run --name api -p 8000:8000 --env USER --env PASSWORD neurobagel/api
+docker run -d --name api -p 8000:8000 --env USERNAME --env PASSWORD neurobagel/api
 ```
-#### Option 2: Build the image using the Dockerfile
+#### Option 3: Build the image using the Dockerfile
 After cloning the current repository, build the Docker image locally:
 ```bash
+source .env # set your environment variables
 docker build -t <image_name> .
-docker run -d --name api -p 8000:8000 --env USER --env PASSWORD neurobagel/api
+docker run -d --name api -p 8000:8000 --env USERNAME --env PASSWORD neurobagel/api
 ```
-For either option, if you wish to also set `DOG_ROOT`, make sure to pass it to the container in the `docker run` command using the `--env` flag.
+
+For Options 2 or 3, if you wish to also set `DOG_ROOT`, make sure to pass it to the container in the `docker run` command using the `--env` flag.
 
 NOTE: In case you're connecting to the McGill network via VPN and you started the container before connecting to the VPN, make sure to configure your VPN client to allow local (LAN) access when using the VPN.
 
```

Comment:
Two things I would add here:

1) I have just started everything for the first time (like you said): now what do I do? is there some link I can click that should now open?
2) It'd be nice to add a line on: "OK, if you run this all for the first time, you will need to create a graph database to be queried (and put data in it). The easiest way is probably to use Stardog-Studio: https://docs.stardog.com/stardog-applications/dockerized_access, point it at `localhost:5820` (unless you changed that) and login with `admin, admin`. Good luck with that"